sdk/src/client/api/block_builder/input_selection/requirement/mod.rs

```diff
@@ -115,8 +108,7 @@ impl InputSelection {
                         self.requirements.push(requirement);
                     }
 
-                    let requirement =
-                        Requirement::Account(*foundry_output.account_address().account_id(), AccountTransition::State);
```

Comment:
So we will not support anchor outputs in the ISA?

Reply:
Correct